x-pack/platform/plugins/shared/stack_connectors/server/connector_types/crowdstrike/index.ts

```diff
@@ -37,7 +40,7 @@ export const getCrowdstrikeConnectorType = (
     secrets: CrowdstrikeSecretsSchema,
   },
   validators: [{ type: ValidatorType.CONFIG, validator: urlAllowListValidator('url') }],
-  supportedFeatureIds: [EndpointSecurityConnectorFeatureId],
+  supportedFeatureIds: [EndpointSecurityConnectorFeatureId, WorkflowsConnectorFeatureId],
```

Comment:
Hey @semd ,
I'm not sure that the connectors that are used for Security Response Actions should be exposed to workflows. These connectors:

- Crowdstrike
- Microsoft Defender for Endpoint
- SentinelOne

are really meant to be used internally by the server code in Security Solution only and not invoked directly, so I'm thinking that `WorkflowsConnectorFeatureId` should not be added to them. 


cc/ @dasansol92 , @ashokaditya , @tomsonpl

Reply:
Thanks for the feedback @paul-tavares.
Do you mean they can't be used for some technical reason? Or some limitation/complexity to call them from workflows? 

cc @talboren @tinnytintin10

Reply:
They can't be used because of the restrictions we impose on top of them. The connectors themselves **do not expose** any APIs via the connectors/actions framework - they include only server-side services that are invoked by our own APIs.

These connectors provide us with an API interface to the 3rd party system and are normally setup with very high level of privileges (api keys). Invoking them from Elastic stack is only supported via our response action APIs so that property RBAC is checked/enforced along with ensuring that response actions are created and property completed. They are not invocable via the Connectors/Sub-action stack framework APIs

Reply:
Okay, I removed the `endpointSecurity` connectors from the supported ones here
https://github.com/elastic/kibana/pull/243042/commits/de0b2b555e435b9e1617939e2caaf1c8c20b3dec and description updated.

Let's discuss how to add support for them in a separate [issue](https://github.com/elastic/kibana/issues/243341)? @shahargl @paul-tavares